lib/smart_proxy_remote_execution_ssh/runners/polling_script_runner.rb

```diff
@@ -126,7 +125,11 @@ def load_event_updates(event_data)
     end
 
     def cleanup
-      run_sync("rm -rf \"#{remote_command_dir}\"") if @cleanup_working_dirs
+      if @cleanup_working_dirs
+        assert_remote_command("rm -rf \"#{remote_command_dir}\"",
```

Comment:
This is still assert, I'd expect it would fail, right?

Reply:
Correct

Reply:
Oh, I misunderstood the comment previously, sorry about this